patitasuy/src/app/pages/profile/profile.component.scss

```diff
@@ -81,16 +63,10 @@
     color: rgba(0, 0, 0, 0.87);
 }
 
-.descriptionState {
-    font-family: 'Roboto';
-    font-style: normal;
-    font-size: 1em;
-    color: #FF9934;
-}
-
 .editButton {
-    --background: #FF9934;
+    --background: rgb(31, 154, 154);
```

Comment:
Replace this with color="secondary" attribute in the button with the editButton class